test_pool/timer_wd/test_w003.c

```diff
@@ -33,6 +33,12 @@ payload(void)
   uint32_t index = val_pe_get_index_mpid(val_pe_get_mpid());
   uint32_t data, ns_wdg = 0;
 
+  if (TEST_NUM != SINGLE_TEST_SENTINEL) {
```

Comment:
What is the intent of this change?
Due to this change, the test will always be skipped.

If this test needs to be skipped at your side, pass -skip 303 with each run

Reply:
Changes to this file have been reverted. They should not have been part of this branch.